docs/configuration/modules/load-flow-action-simulator.md

```diff
@@ -23,9 +23,9 @@ modify the structure of the network.
 Set the `ignore-pre-contingency-violations` to `true` to ignore the the pre-contingency violations and continue the
 simulation even if there are still violations after the pre-contingency simulation.
 
-## load-flow-factory
-Use the `load-flow-factory` property to define the `com.powsybl.loadflow.LoadFlowFactory` implementation to use for the
-simulation.
+## load-flow-name
```

Comment:
I don't agree: deprecated properties can still be used (in your example, `availableCore` can still be used and understood). Here `load-flow-factory` will not work anymore, the cases are not the same and we should probably discriminate them to prevent confusion. I think @jonenst 's idea of versioning documentation pages might be useful here... Failing that, another section `Removed properties` might work.

Reply:
added ```Removed properties```